util/generate-ami-list.py

```diff
@@ -215,6 +218,9 @@ def update_amis_txt(amis_txt_file, amis):
     group2 = parser.add_argument_group("Retrieve instances from EC2 searching by cookbook and node git reference")
     group2.add_argument("--cookbook-git-ref", type=str, help="cookbook git hash reference", required=False)
     group2.add_argument("--node-git-ref", type=str, help="node git hash reference", required=False)
+    group2.add_argument(
```

Comment:
minor: `type=str` is the default so we can remove it from all the arguments.
minor2: Why did you add `(optional)` only for this parameter?

Reply:
because you could not specify it and the latest will be taken